modules/ROOT/pages/to-find-info.adoc

```diff
@@ -30,54 +13,137 @@ Exchange Search looks for terms in:
 * The content of the asset portal pages
 * xref:to-find-info.adoc#searching-api-specifications[Properties defined in RAML and OAS API specifications]
 
-You can combine search terms with the filters listed in All Types.
+In Exchange you can enter search text to search for assets, click *All Types* to set a filter for specific asset types, or combine search text with the filters listed in the type menu.
+
+<<index.adoc#content-types,Content Types>> lists the different types of asset.
+
+You can also search by clicking category or tag elements. This creates a filtered search you can use to find similar assets.
+
+The original Exchange search experience is available in Anypoint Platform Private Cloud Edition, MuleSoft Government Cloud, the US cloud, and the EU cloud. The new Exchange search experience is available in the US cloud and the EU cloud.
+
+New Exchange search experience:
```

Comment:
I'm thinking we need a new page to put the information specific to the new search experience. Other sections have information about old search and could be confusing. For example "Search Separators" "Phrase Search Separators" apply to old search only, and we have a "== Search by Tags, Categories, and Custom Fields in the Original Exchange Search Experience" section where we make a distinction for tags, categories and custom fields.

I would suggest to keep this page with information about old search and add a new one for new search where we can more clearly explain the differences.

Reply:
If I understand correctly, we do plan to remove the old search experience eventually and have one search system everywhere. In that case, I think it's better to describe both systems on one page, so that when we go to one system, we don't have to delete a page from the docs.

We try to avoid deleting docs pages and making new redirects, since it hurts our search engine rankings and surprises any users who have bookmarks to the old page.

I agree that we should make the difference between old and new search clearer and more organized.

As I reviewed your comment, I found that the introduction text still had some info about types in the old search, and moved that to a section about the old search.

Currently this is the outline:

[general info, including info about where old and new search are available, and basic tasks with new search]
* Save a Search [both old and new]
* Searching API Specifications [both old and new]
* Search for Phrases in the Original Exchange Search Experience [old]
  * Search Separators [old]
  * Phrase Search Separators [old]
* Search by Tags, Categories, and Custom Fields in the Original Exchange Search Experience [old]
  * Search by Tags [old]
  * Search by Categories [old]
  * Search by Custom Fields [old]
* See Also [both old and new]

In the navigation menu on the right side, only the top level headers appear:

* Save a Search [both old and new]
* Searching API Specifications [both old and new]
* Search for Phrases in the Original Exchange Search Experience [old]
* Search by Tags, Categories, and Custom Fields in the Original Exchange Search Experience [old]
* See Also [both old and new]

To make this clearer, I have added text in every section either in the title or the first sentence explaining whether that section is for the old search, the new search, or both. I have also re-organized the headings:

[general info, including info about where old and new search are available]
* Tags, Categories, and Ranking in the New Exchange Search Experience [new]
* Searching in Organizations [both old and new]
* Save a Search [both old and new]
* Searching API Specifications [both old and new]
* Search for Phrases in the Original Exchange Search Experience [old]
* Search by Types, Tags, Categories, and Custom Fields in the Original Exchange Search Experience [old]
* See Also [both old and new]

Please review the new draft and tell me what you think.

Reply:
Got it, I was not considering that we would need to delete the new page after making it the default experience 👍 .
The changes look good, thanks!